src/styles/AdminDashboardHome/index.less

```diff
@@ -36,6 +38,20 @@ h2 {
   margin-left: 10px;
 }
 
+.see-all-link {
```

Comment:
after seeing the inline styles, I'm happy that this code is written inside a stylesheet.